app/assets/stylesheets/scaffold.scss

```diff
@@ -39,40 +43,43 @@ h6 {
   margin: 5px 0;
 }
 
-.top-bar {
-  background-color: #660000;
-  color: #eee;
-  padding: 10px 15px;
-  border-bottom: 5px solid #000000;
-}
-
 main {
   padding: 0 20px;
 }
 
 a {
-  text-decoration: none;
   color: inherit;
+  text-decoration: none;
 }
 
 a:hover {
-  text-decoration: underline;
   color: #00008b;
+  text-decoration: underline;
+}
+
+// sitewide top header
+
+.top-bar {
+  background-color: $color-red;
+  border-bottom: 5px solid $color-black;
+  color: $color-white;
+  margin-bottom: 20px;
+  padding: 10px 20px;
 }
 
 header aside li {
   display: inline;
   float: right;
   font-size: 12px;
-  padding: 16px 8px;
+  padding: 31px 8px;
 }
 
-.logo {
-}
+// sitewide user bars
 
 .user-bar li {
   display: inline;
   float: right;
   font-size: 14px;
-  padding: 20px 8px;
+  margin-top: 14px;
+  padding-right: 8px;
 }
```

Comment:
It looks like this file has some mixed intent! I would create a header and footer file where that css can get stored, and leave the site-wide styling for individual element types here :)